client/helpers.go

```diff
@@ -301,6 +307,24 @@ func isNotFoundError(err error) bool {
 	return false
 }
 
+// IsAccessDeniedError checks if api error should be classified as a permissions issue
+func (c *Client) IsAccessDeniedError(err error) bool {
+	if isAccessDeniedError(err) {
+		c.logger.Warn("API returned \"Access Denied\" error ignoring it...", "error", err)
```

Comment:
Access Denied -> AccessDenied ?

Reply:
Actually we don't know if it's `Access Denied` or some other error in the `accessDeniedErrorStrings` map, so log line not that accurate.

Reply:
Updated the message